### ISSUE TITLE
created version 2.0

### ISSUE THREAD

**Issue by darknessblade** [sparkfun/Pokedex#18]
hello guys.

i have created a newer better version of the pokedex casing.
will upload the files as soon as i have fully completed it. as in a functional state
(if the owner of the github page allows me to upload the files, that would be great) (i have currently a beta version complete) (not sure if its 100% functional/working as intended) (it fits ALL phones up to 5.5" without the need of complex changes). (since i used a tray mechanism to put in the phone)
## 

features:
## 

still the same size (thus no need for a bigger printer)
## 

phone swappable. (made a slide in tray for the phone) (maximum supportable size 140x 70 mm) (5"phones)
its also possible to use a 5.5"phone (you will lose some part of the screen and speakers) (or the powerfunction has to be sacrifised) (the screen los is bare minimum so should not inflict any pokemon capture issue's)
## 

possible powerbank hotswap (swapping 2000mah powerbanks) (bar/cilinder versions) (will post maximum combiable size powerbanks soon)
## 

smoothed out the layers.
## 

other features will be added soon coming soon.

**Comment by jamesalmeida** [comment]
Looking forward to seeing this! Once this is live we can mark this other one as done: https://github.com/sparkfun/Pokedex/issues/8

**Comment by darknessblade** [comment]
since i have no push access i cannot upload the designs yet.
if possible i will start printing the _functional_ prototype.

since my design features a bay like phone case, the phonecase-bay can be adapted to fit almost any phone up to 5,5"  (based on my phone jiayu S3)

---

since slicer programs may have issues with corrupt lines (wich ruin the entire print) it can take some days to fully print the design.

the total weight is about 300GR (using full support) (material cost would be about 7-12$)

**Comment by darknessblade** [comment]
since i have no push access i cannot upload the designs yet.
if possible i will start printing the *functional* prototype.

since my design features a bay like phone case, the phonecase-bay can be adapted to fit almost any phone up to 5,5"  (based on my phone jiayu S3)
------------------
since slicer programs may have issues with corrupt lines (wich ruin the entire print) it can take some days to fully print the design.

the total weight is about 300GR (using full support) (material cost would be about 7-12$)

**Comment by darknessblade** [comment]
since i have no push access i cannot upload the designs yet.
if possible i will start printing the *functional* prototype.

since my design features a bay like phone case, the phonecase-bay can be adapted to fit almost any phone up to 5,5"  (based on my phone jiayu S3)
------------------
since slicer programs may have issues with corrupt lines (wich ruin the entire print) it can take some days to fully print the design.

the total weight is about 300GR (using full support) (material cost would be about 7-12$)

**Comment by a5m0** [comment]
@darknessblade will your design block side buttons of phones?

**Comment by darknessblade** [comment]
it originally blocks the side buttons, but it should be possible to make it in such a way that it can be pushed trough a tilted button extension.  (right side only for now)

with the original design that is not possible since it would be a structural design disaster if you would print it that way. since my design features a bay for the phone its more adaptable for such problems

**Comment by jamesalmeida** [comment]
@darknessblade you could fork this repo and start your own to upload your files too.

**Comment by darknessblade** [comment]
sadly forking a project only works for code, not sketchup files, (i can add the code in the file but it can get corrupted then)

**Comment by darknessblade** [comment]
sadly forking a project only works for code, not sketchup files, (i can add the code in the file but it can get corrupted then)

**Comment by bboyho** [comment]
Hi,

I happened to come across this issue when checking out this repository. Sorry for the delay in reply. The project files were released to the public for those interested in adjusting the models and making their own case. It looks like you might have an updated version but I am not sure if it was ever [uploaded in your repo](https://github.com/darknessblade/Pokedex). We could try to do a pull request to include your files but it sounds like it might corrupt the files. At the least, we could link to your repository in the README.md if the files are in the repo. I'll be closing this issue out but we can re-open it if necessary.
